cluster/gce/trusty/kube-manifests/kube-apiserver.manifest

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
I removed "/usr/share/ssl", "/usr/ssl", "/usr/lib/ssl" and "/usr/local/openssl". In addition to it, I also changed mount path /srv/kubernetes to /etc/srv/kubernetes. 

In this PR, these two manifests are the only duplicated configs. An option is to make the parts mentioned above in these two files configurable. More generally, I think we should remove the  copies in dirs cluster/gce/<coreos and trusty>/kube-manifests, and all distros only use the manifests copied from the real source cluster/saltbase/salt/. That is why I added a TODO in build/common.sh (in this PR) saying we should unify the generation of manifests.